docs/user-guide/explore/trace-view.md

```diff
@@ -0,0 +1,53 @@
+---
+sidebar_position: 4
+title: Traces in Explore
+description: View and investigate distributed traces directly in Explore
+image: https://dytvr9ot2sszz.cloudfront.net/logz-docs/social-assets/docs-social.jpg
+keywords: [logz.io, trace, traces, explore, traces in explore, log analysis, observability]
+---
+
+
+Trace View lets you to view and analyze distributed traces directly in Explore, offering a comprehensive understanding of your system's activity. It helps correlate logs with traces to identify bottlenecks, pinpoint errors, and understand your application's complete execution path. 
+
+## Configure Trace view
+
+To enable Trace View in Explore, you’ll need:
+
+* An active Logz.io Tracing account.
+* A `trace_id` value for all traces. Most OpenTelemetry compliant tracing libraries automatically include `trace_id`. Use Logz.io’s collector to send your traces.
```

Comment:
* A `traceID` value for all traces. Most OpenTelemetry compliant tracing libraries automatically include `traceID`. Use Logz.io’s collector to send your traces.
* A `trace_id` value for all logs correlated with traces. Most OpenTelemetry compliant logging libraries automatically include trace context feature to populate it. Use Logz.io’s SDKs to send your trace context propagated logs.